docs/internal/publishing_docs_using_mkdocs.md

```diff
@@ -41,6 +41,9 @@ nav:
 
 ## Testing docs locally
 
-We've got a dockerized setup for easily testing docs in local environment. Simply run `docker-compose up` in the docs root folder. The docs will be available on `localhost:80`.
+We've got a dockerized setup for easily testing docs locally. Simply run `make
```

Comment:
I'd suggest breaking the line after or before `make docs-serve-dev`.

Reply:
Done